src/utils/transactionUtils/transactionUtils.ts

```diff
@@ -572,3 +576,15 @@ export function getAssociatedNames(
     })
     .filter((n) => !!n);
 }
+export async function validateTTLSeconds(ttl: number): Promise<void> {
```

Comment:
nit: this isn't an async function so `async` and `Promise` are not needed 😄

Reply:
yeah this was simply to make it compatible with the Promise<any> return type

Reply:
changed to a manual promise